src/workers/WorkerRunner.ts

```diff
@@ -0,0 +1,42 @@
+import { setIntervalAsync } from 'set-interval-async/dynamic';
+import { logger } from '../logger';
+import { IWorker } from './framework';
+
+export type RunOptions = { resyncFrom?: number; fetchInterval?: number };
+
+export default class WorkerRunner {
+  workers: {
```

Comment:
```suggestion
  workers: Array<{...
```
nit: make it more obvious.